### PR TITLE
Correction of  documentation

### PR DESCRIPTION
**Pull request by MadayAlcala**

### CODE REVIEW

**Review by JosueRodriguezGarcia — APPROVED** [review]

File: src/main/java/com/foundation/trello/model/Card.java [hunk 1]
```diff
@@ -2,8 +2,46 @@
...
  */
 public class Card {
-    //Card class for jsonConverter.
+    private String id;
+    private String name;
+
+    /**
+     * This method is used to get id.
+     *
+     * @return a string.
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/model/Card.java]: Please explain the content and use of String

File: src/main/java/com/foundation/trello/model/Card.java [hunk 2]
```diff
@@ -2,8 +2,46 @@
...
+     * @param id the id parameter defines of input id.
+     */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * This method is used to get name.
+     *
+     * @return a string.
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/model/Card.java]: Please explain the content and use of String

File: src/main/java/com/foundation/trello/model/List.java [hunk 3]
```diff
@@ -2,8 +2,46 @@
...
+     * @param id the id parameter defines of input id.
+     */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * This method is used to get name.
+     *
+     * @return a string.
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/model/List.java]: Please explain the content and use of String

File: src/main/java/com/foundation/trello/util/ReadConfiguration.java [hunk 4]
```diff
@@ -41,6 +43,9 @@ private Properties readConfigurationFile() {
     }
 
     /**
+     * This method ensures that only one instance is created according
+     * to the builder pattern.
+     *
      * @return an instance.
```
Comment (aburgosu) [inline src/main/java/com/foundation/trello/util/ReadConfiguration.java]: Please explain the content and use of returned instance

File: src/main/java/com/foundation/trello/model/Card.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/model/Card.java]: Solved.

File: src/main/java/com/foundation/trello/model/Card.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/model/Card.java]: Solved.

File: src/main/java/com/foundation/trello/model/List.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/model/List.java]: Solved.

File: src/main/java/com/foundation/trello/util/ReadConfiguration.java (hunk 4 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/ReadConfiguration.java]: Solved.

**Review by aburgosu — APPROVED** [review]

**Pull request merged**